azerothcore/azerothcore-wotlk
# fix(DB/creature_addon): Attach auras to Winter Revelers to correct models.

Co-authored by: @cala, @AnonXS



## Changes Proposed:
- Adds auras to [Winter Revelers](https://www.wowhead.com/wotlk/npc=15760) to correct their models.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/4561
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/14257

## SOURCE:

cmangos, both [TBC](https://github.com/cmangos/tbc-db/commit/fcef33fa559013bb5718c7fe83a460108f1031b6) and [Classic](https://github.com/cmangos/classic-db/commit/c2b32b5bedbe30121140e873ab06ac68c272ced7).

## Tests Performed:

- [Tested in-game](https://github.com/azerothcore/azerothcore-wotlk/pull/14366#issuecomment-1364638301).

## How to Test the Changes:

1. Go to each Winter Reveler.
2. Observe their model, if it looks wrong, then it's wrong.

## Known Issues and TODO List:

- [X] ~There are a couple creatures ([109159, 109160](https://user-images.githubusercontent.com/38561765/209459305-5dc642c6-007d-485a-8a91-6a47bdcaa0ec.png)) that are on the Northrend map that also need to have their auras sniffed.~ Turns out the creatures in question don't exist on Wrath Classic in the spot they do in AC. I'm calling that a separate issue outside the scope of this PR.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
